ydb/core/tx/schemeshard/schemeshard_export__create.cpp

```diff
@@ -311,7 +312,18 @@ struct TSchemeShard::TExport::TTxProgress: public TSchemeShard::TXxport::TTxBase
             << ", txId# " << txId);
 
         Y_ABORT_UNLESS(item.WaitTxId == InvalidTxId);
-        Send(Self->SelfId(), BackupPropose(Self, txId, exportInfo, itemIdx));
+        if (item.SourcePathType == NKikimrSchemeOp::EPathTypeTable) {
+            Send(Self->SelfId(), BackupPropose(Self, txId, exportInfo, itemIdx));
+        } else if (IsIn({
+                NKikimrSchemeOp::EPathTypeView
+            }, item.SourcePathType
+        )) {
```

Comment:
Очень странное форматирование... Вполне влезает в одну строку:
```suggestion
        } else if (IsIn({NKikimrSchemeOp::EPathTypeView}, item.SourcePathType)) {
```

Либо выдели условие отдельно в функцию/переменную.

Reply:
Это я раньше времени заложился на будущее, чтобы diff при добавлении нового схемного объекта был однострочным. [Переписал](https://github.com/ydb-platform/ydb/blob/7fff44680723d7c69599776392428d7286c23865/ydb/core/tx/schemeshard/schemeshard_export__create.cpp#L326)